packages/firestore/src/local/memory_persistence.ts

```diff
@@ -228,6 +228,7 @@ export class MemoryEagerDelegate implements ReferenceDelegate {
     return cache
       .getMatchingKeysForTargetId(txn, queryData.targetId)
       .next(keys => {
+        console.log('Found matching keys ' + keys.size);
```

Comment:
Shouldn't tslint yell about this? (and throughout the rest of this PR)

Reply:
Uhm. Sorry. I had a bunch of GitHub trouble where it wouldn't show my latest changes. Looks like I rebasing my stashed changes from last week solved my troubles :/